casepro/msgs/models.py

```diff
@@ -303,7 +303,7 @@ class Message(models.Model):
 
     type = models.CharField(max_length=1)
 
-    text = models.TextField(max_length=640, verbose_name=_("Text"))
+    text = models.TextField(verbose_name=_("Text"))
```

Comment:
As I said in dev list discussion - we can increase this but we should have *a* hard limit. External systems need to know what the absolute longest message they might receive this. Do you guys have a need to be sending anything longer than say 800 (5 concatenated 7bit SMSs) ?